src/main/java/seedu/address/logic/parser/PictureCommandParser.java

```diff
@@ -37,8 +36,8 @@ public PictureCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, PictureCommand.MESSAGE_USAGE));
         }
 
-        Path fileLocation = ParserUtil.parseFileLocation(argMultimap.getValue(PREFIX_FILE_LOCATION).get());
+        Picture picture = new Picture(argMultimap.getValue(PREFIX_FILE_LOCATION).get());
```

Comment:
You don't want to create a new picture using illegal values (illegal argument exception isn't handled elegantly for the user), so have one more if clause that catches `!isValidPicture(fileLocation)` before this as above, and throw a ParseException. This way, the flow won't break when there is an illegal path.